File: ProgrammingLanguage/Python/HelloWorld/Fab.py
#! /usr/bin/env python
# -*- coding:utf-8 -*-

def fab(n) :
    a, b = 0, 1
    while n :
        print a,
        a, b = b, a+b
        n -= 1
    
def main() :
    fab(10)
    

if __name__ == "__main__" : main()

File: ProgrammingLanguage/Java/HelloWorld/Fab.java
public class Fab 
{
    public static int fab(int n) {
        int a = 0, b = 1;
        for (int i = 0; i < n; ++i) {
            int tmp = a;
            a = b;
            b = tmp + b;
        }
        return a;
    }

    public static void main(String[] args) 
    {
        for (int i = 0; i <= 20; ++i) {
            System.out.println(i + " " + fab(i));
        }
    }
}
